app/Http/Middleware/HandleInertiaRequests.php

```diff
@@ -7,18 +7,6 @@
 
 class HandleInertiaRequests extends Middleware
 {
-    /**
-     * Determines the current asset version.
-     *
-     * @see https://inertiajs.com/asset-versioning
-     * @param  \Illuminate\Http\Request  $request
-     * @return string|null
-     */
-    public function version(Request $request)
-    {
-        return parent::version($request);
-    }
-
```

Comment:
Good point ! I didn't see any value in the function, but maybe I'm missing something, maybe @WindOfRussia can help us out since he had experience with Inertia

Reply:
Yes, this is a required function let's keep the function and mark any issues in Sonar as a False-positive

Reply:
This is a stub, even if it looks off, better to keep it

Reply:
@alecadub You can mark it as a false positive in Sonarcloud and undo the change